config/routes.rb

```diff
@@ -60,6 +60,7 @@
   end
 
   namespace :comparisons do
+    resources :annual_electricity_costs_per_pupil, only: [:index]
```

Comment:
yeah we should probably sort these, I've just been adding to the bottom :grimacing:

Reply:
The route generator seems to put these at the top of the namespace by default.. (now it is working - see https://github.com/Energy-Sparks/energy-sparks/pull/3472)